metals/src/main/scala/scala/meta/internal/builds/ShellRunner.scala

```diff
@@ -130,3 +133,80 @@ class ShellRunner(
   }
 
 }
+
+object ShellRunner {
+
+  /**
+   * Note: if returning a [[Future]] is more suitable, all you
+   * need to do is to uncomment the commented code.
+   */
+  def quickRunWithShortStringResult(
+      //        commandRun: String,
+      args: List[String],
+      directory: AbsolutePath,
+      redirectErrorOutput: Boolean,
+      additionalEnv: Map[String, String] = Map.empty,
+      processOut: String => Unit = scribe.info(_),
+      processErr: String => Unit = scribe.error(_),
+      propagateError: Boolean = false,
+      maybeJavaHome: Option[String]
+
+      //       logInfo: Boolean = true
+  ):
+  // Future[
```

Comment:
Let's remove these if not needed. I think it should be fine to run it synchronously since it should be quick. Just let's name the method explicitely.